src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -180,9 +180,9 @@ class UserService(protected val userInfo: UserInfo, dataSource: DataSource, prot
     } flatMap (_ match {
       case Some(g) => Future.successful(g)
       case None => createGroupInternal(allUsersGroupRef, txn).recover {
-        // this case is where the group was not in our db but already in google, the recovery code makes the assumption
-        // that createGroupInternal saves the group in our db before creating the group in google so loadGroup should work
-        case t: HttpResponseException if t.getStatusCode == StatusCodes.Conflict.intValue => containerDAO.authDAO.loadGroup(allUsersGroupRef, txn).get
```

Comment:
Yes this assumption seemed to not always be right (at least in this case), it could also fail to create the google group and then just create the rawls group - im not sure what would happen if this occurred, but the change appears to fix the current problem